window_ops/rolling.py

```diff
@@ -160,6 +160,30 @@ def rolling_min(x: np.ndarray,
                 min_samples: Optional[int] = None) -> np.ndarray:
     return _rolling_comp(_lt, x, window_size, min_samples)
 
+@njit
+@_rolling_docstring
+def rolling_sum(input_array: np.ndarray, window_size: int, min_samples: int | None = None) -> np.ndarray:
```

Comment:
This is python 3.10+ syntax
```suggestion
def rolling_sum(input_array: np.ndarray, window_size: int, min_samples: Optional[int] = None) -> np.ndarray:
```